Examples/KeyboardSupport-iOSExample/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  KeyboardSupport-iOSExample
+//
+//  Created by Earl Gaspard on 7/30/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+import KeyboardSupport
+
+class ViewController: UIViewController, KeyboardRespondable {
+
+    // IBOutlets
+    @IBOutlet private var scrollView: UIScrollView!
+    @IBOutlet private var textField1: UITextField!
+    @IBOutlet private var textField2: UITextField!
+    @IBOutlet private var textField3: UITextField!
+    @IBOutlet private var textView: UITextView!
+    
+    // KeyboardScrollable
+    var keyboardScrollableScrollView: UIScrollView?
+    var keyboardWillShowObserver: NSObjectProtocol?
+    var keyboardWillHideObserver: NSObjectProtocol?
+    
+    // KeyboardNavigator
+    private var keyboardNavigator: KeyboardNavigator?
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        textView.delegate = self
+        
+        // KeyboardDismissable setup
+        setupKeyboardDismissalView()
+        
+        // KeyboardScrollable setup
+        keyboardScrollableScrollView = scrollView
+        setupKeyboardObservers()
+        
+        // KeyboardToolbar setup
+        let keyboardToolbar = KeyboardToolbar(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: 50))
```

Comment:
I'm guessing it's because it's in `viewDidLoad` so the frame is off, so maybe we can look to move this to `viewDidLayoutSubviews`? Possibly even instantiate and configure with a `.zero` frame and then adjust frame only on `viewDidLayoutSubviews`?

Reply:
The view controller's `view` should have the correct frame in `viewDidLoad()` though, right? I know sometimes Auto Layout hasn't kicked in for some of the subviews, but the parent view should be okay to use I would think. I tested in the simulator and couldn't see a difference.

Reply:
It can, but isn't guaranteed until `viewDidLayoutSubviews` (well I guess if we just need the view's frame, we could use `viewWillLayoutSubviews`. Usually in `viewDidLoad` the frame will be whatever size the view is in the storyboard/nib or .zero if it's in code.

It's also possible I'm overcomplicating the reasoning here and it's just an oversight that `UIScreen` is being used.

Reply:
Could not using UIScreen could result in an incorrect Toolbar width when a view inside a multitasking app displays the keyboard?

Reply:
I did not see any visual problems using `view.bounds.width`. Also there were no visual issues using multitasking that I could tell.
For simplicity I am going to leave the initialization of `keyboardToolbar` in `viewDidLoad`.